RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -995,7 +995,51 @@ public void TestNestedVbFormModuleConfigWithProperty()
             AssertTree(parseResult.Item1, parseResult.Item2, "//moduleConfigProperty", matches => matches.Count == 1);
         }
 
-        
+
+        [Test]
+        public void GermanStyleFloatingPointsInFormsPart()
```

Comment:
Should we have another test that has both formats in the same form definition to assert that it works for either formats?

Reply:
I see no reason why this should fail, but I will add a test nonetheless.